documentation/docs/getting_started/python.mdx

```diff
@@ -20,14 +20,28 @@ import SafePasswordStorage from '../_admonitions/_warning_safe_password_storage.
 * [setuptools-rust](https://pypi.org/project/setuptools-rust/) >= 0.10.2
 * Make sure you have fulfilled the [Rust Getting Started](rust.mdx) requirements to compile the binding.
 
+:::info Windows Requirements
```

Comment:
Would it make more sense to include it to the `rust.mdx` since it's required also for all bindings this way?

Reply:
True that 🤔 
Will update the PR later on 👍